### PR TITLE
(feat): add google colab env config

### PR DESCRIPTION
**Pull request by sjmignot**

### CODE REVIEW

File: notebook-setup.mdx [hunk 1]
```diff
@@ -75,7 +75,66 @@ The Chalk client can then be initialized referencing the variables directly
 ```python notebook.ipynb
 from chalk import ChalkClient
```
Comment (emarx) [inline notebook-setup.mdx:76]: i thtitnk it's from chalk.client?

Comment (sjmignot) [reply notebook-setup.mdx:76]: Yup, nice catch

File: notebook-setup.mdx [hunk 2]
```diff
@@ -57,7 +57,7 @@ You can then initialize Chalk by running the following code in a cell:
 import os
 from chalk import ChalkClient
```
Comment (sjmignot) [inline notebook-setup.mdx:58]:
```suggestion
from chalk.client import ChalkClient
```

File: notebook-setup.mdx (hunk 1 above)
Comment (sjmignot) [inline notebook-setup.mdx:76]:
```suggestion
from chalk.client import ChalkClient
```

File: notebook-setup.mdx [hunk 3]
```diff
@@ -73,9 +73,68 @@ Select `Python 3.10` as the image under **Environment > Compute Profile**.
...
 
-client = ChalkClient()
+client = ChalkClient(branch="new-feature")
+```
+
+### Vertex & Google Colab Enterprise
+
+On Google Colab Enterprise, you can set up secrets using the Google Secret Manager.
+Add secrets named `chalk-client-id` and `chalk-client-secret` to Google Secret Manager
+for your Chalk client id and client secret.
```
Comment (hugokitano) [inline notebook-setup.mdx:85]:
```suggestion
that correspond to your Chalk client id and client secret.
```

**Review by hugokitano — APPROVED** [review]

File: notebook-setup.mdx [hunk 4]
```diff
@@ -73,9 +73,68 @@ Select `Python 3.10` as the image under **Environment > Compute Profile**.
...
+from google.cloud import resourcemanager_v3 as rm
+
+from chalk.client import ChalkClient
+
+# Create the Secret Manager client.
+PROJECT_NAME = os.environ['GOOGLE_CLOUD_PROJECT']
+
+# These should match the names given to your secrets in Google Secret Manager
+client_id_name = "chalk-client-id"
+client_secret_name   = "chalk-client-secret"
```
Comment (sjmignot) [inline notebook-setup.mdx:111]:
```suggestion
client_secret_name  = "chalk-client-secret"
```

**Pull request merged**